composer/utils/file_helpers.py

```diff
@@ -415,13 +415,11 @@ def maybe_create_remote_uploader_downloader_from_uri(
                                   's3 or one of the supported RemoteUploaderDownloader object stores')
 
 
-def get_file(
-    path: str,
-    destination: str,
-    object_store: Optional[Union[ObjectStore, LoggerDestination]] = None,
-    overwrite: bool = False,
-    progress_bar: bool = True,
-):
+def get_file(path: str,
```

Comment:
tiny nit, can you add the trailing commas back to these so the formatting doesn't change?

Reply:
fixed!